strongbox-security/strongbox-security-api/pom.xml

```diff
@@ -127,6 +127,7 @@
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>3.2.0</version>
```

Comment:
This is not the way to do this. You have to update the `strongbox-parent` project. You can have at [this article](https://strongbox.github.io/developer-guide/upgrading-dependencies-and-plugins.html).